gui/test/setup/main.js

```diff
@@ -2,3 +2,4 @@ const log = require('electron-log');
 
 log.transports.console.level = false;
 log.transports.file.level = false;
+log.transports.rendererConsole = null;
```

Comment:
FYI this is the IPC transport that broadcasts logs from main -> renderer console. The `console` transport level does not affect it, so we have to disable it to avoid the logs from appearing in tests.

Related: https://github.com/megahertz/electron-log/issues/138